composer.json

```diff
@@ -21,7 +21,8 @@
     ],
     "require": {
         "php": "^7.0",
-        "phpunit/phpunit": "^6.0.5|^7.0"
+        "phpunit/phpunit": "^6.0.5|^7.0",
+        "ext-mbstring": "*"
```

Comment:
Is this necessary? Do we really need to rely on an extension for this? What's wrong with `strlen`?